hq_superset/utils.py

```diff
@@ -37,3 +47,51 @@ def get_role_name_for_domain(domain):
     # Prefix in-case domain name matches with known role names such as admin
     # Same prefix pattern as schema only by coincidence, not a must.
     return f"{DOMAIN_PREFIX}{domain}"
+
+
+def get_column_dtypes(datasource_defn):
+    """
+    Maps UCR column data types to Pandas data types.
+
+    See corehq/apps/userreports/datatypes.py for possible data types.
+    """
+    # TODO: How are array indicators handled in CSV export?
+    pandas_dtypes = {
+        'date': 'datetime64[ns]',
+        'datetime': 'datetime64[ns]',
```

Comment:
Does this keep the timezone info intact in the final Postgres DB?

Reply:
The "Specify date/datetime columns" commit actually might ... by accident!

When I was writing this, I got it into my head that Dataframes don't support timezones, [but that's not true](https://pandas.pydata.org/pandas-docs/stable/user_guide/basics.html#attributes-and-underlying-data): "NumPy doesn’t have a dtype to represent timezone-aware datetimes" not Pandas.

I'll test timezone-aware UCR datetimes, and change the spelling of these so it's less misleading that these are Pandas data types. (You'll see in the "Specify date/datetime columns" commit that we don't specify data types for dates and datetimes.)